docs/secure/Idp_integration.md

```diff
@@ -0,0 +1,142 @@
+
+            ![!image](../images/OKTA_admin_console.png)
+
+        4. Set the following fields:
+
+            a. App integration name - any value
+
+            b. Sign-in redirect URIs - `<EVEREST_URL>/login-callback`
+
+            c. Sign-out redirect URIs - `<EVEREST_URL>`
+
+            d. Click **Save**.
+
+            e. Copy the `clientID` of the created app.
+
+            f. Navigate to **Security → API → Authorization Servers** and copy the `issuerURL` you’d like to use for the Everest authorization. 
+
+            !!! note "Note"
+                Okta allows the use of HTTP for development purposes and in cases where the Admin explicitly permits it.
+
+            ![!image](../images/sso_aap_integration.png)
+
+
+
+## Configure OIDC on Percona Everest side
+
+
+You can configure OIDC via Percona Everest CLI:
+
+    everestctl settings oidc configure --issuer-url=http://url.com --client-id=id2342
+
+??? info "What's happening under the hood?"
+
+    This command stores the updated configuration in k8s and restarts the Everest deployment.
+
+    1. **Store Configuration**
+
+        The OIDC settings are stored in the `everest-settings` ConfigMap, along with other settings, in the following format:
+
+        ```sh
+        apiVersion: v1
+        kind: ConfigMap
+        metadata:
+            name: everest-settings
+        data:
+            data:
+            oidc.config: |
+                issuerUrl: <your OIDC provider URL>
+                clientId: <your OIDC provider client ID>
+        ```
+    
+        The Everest user should not directly interact with the `everest-settings` ConfigMap. Use the CLI command to set up the OIDC config.
+
+    2. **Restart Percona Everest**
+
+        OIDC configuration must be in place for Percona Everest to start the API Server, allowing the server to utilize the OIDC provider's validation. This means that after setting up the OIDC configuration, the Percona Everest API Server needs to be restarted.
+
+        The CLI command typically takes approximately 15 seconds to execute. It waits for the Everest Deployment to be up again before exiting successfully.
+
+    
+        !!! info "Important"
+
+            - The restart only impacts the Everest UI and API. Database clusters are not affected.
+
+            - The restart results in the loss of the port-forwarding connection. If you had port-forwarding enabled on your machine to access Percona Everest UI and API, you will need to set it up again.
+
+
+## API to get the OIDC settings
+
+
+The OIDC configuration settings can be retrieved along with other Everest settings using the following API:
+
+```sh
+GET /v1/settings
+```
+
+??? example "Example"
+
+        {        
+           
+            "oidcConfig": {
+            
+            "clientId": "id1234",
+            
+            "issuerURL": "https://url.com"
+        
+            }
+    
+        }
+
+
+For more information, see the [Open api spec](docs/spec/openapi.yml).
```

Comment:
I don't think we should add this section to the docs, @recharte wdyt? The end users don't need the API, because the UI does all the job for them. Maybe it would be better to add a section of what will happen next after the oidc is configured successfully - that the users should go to the Everest login page and they should see that "login with SSO" button.

Reply:
I agree, API has it's own separate docs I don't think it makes sense to have it here.

Reply:
Fixed.